esrally/metrics.py

```diff
@@ -67,8 +67,8 @@ def delete(self, index, id):
     def get_index(self, name):
         return self.guarded(self._client.indices.get, name=name)
 
-    def create_index(self, index, body=None):
-        return self.guarded(self._client.indices.create, index=index, body=body, ignore=400)
+    def create_index(self, index):
+        return self.guarded(self._client.indices.create, index=index, ignore=400)
```

Comment:
`400` is `BAD_REQUEST`. I don't see the scenario when this is useful, but I'm keeping it.

Reply:
So I looked back and in my changes a while ago, I accidentally deleted a comment about 400:
`# ignore 400 cause by IndexAlreadyExistsException when creating an index`

```
https://github.com/elastic/rally/commit/9cd54a4931a77ec37329017e72f4f3343a2feb71#diff-94fb4ab5720523175cf3c1da3e45d24595bcac1988a46dba2e902d4645eb6312R94
```
Maybe we can add it back?

Reply:
Thank you! Somehow I expected a different code for `resource_already_exists_exception`. I've added comments back in https://github.com/elastic/rally/pull/1747/commits/5c75588f5dfdbe5baa15ddd6f83b449b50a24e1c.